iec61850serversimulator/src/main/java/com/cgi/iec61850serversimulator/Scheduler.java

```diff
@@ -15,98 +16,106 @@
  */
 public class Scheduler {
 
-	private ScheduledThreadPoolExecutor executor;
-	private List<ScheduledFuture<?>> scheduledFutures;
-	private TimeCalculator timeCalculator;
-	private SwitchingMomentCalculator switchingMomentCalculator;
-	private static final Logger logger = LoggerFactory.getLogger(Scheduler.class);
-	private Device device;
+    private ScheduledThreadPoolExecutor executor;
+    private List<ScheduledFuture<?>> scheduledFutures;
+    private TimeCalculator timeCalculator;
+    private SwitchingMomentCalculator switchingMomentCalculator;
+    private static final Logger logger = LoggerFactory.getLogger(Scheduler.class);
+    private Device device;
 
-	public Scheduler(Device device) {
-		ScheduledThreadPoolExecutor executor = (ScheduledThreadPoolExecutor) Executors.newScheduledThreadPool(1);
+    public Scheduler(Device device) {
+        ScheduledThreadPoolExecutor executor = (ScheduledThreadPoolExecutor) Executors.newScheduledThreadPool(1);
```

Comment:
Easier: this.executor = (ScheduledThreadPoolExecutor) Executors.newScheduledThreadPool(1);
If you do so, you can delete line 35.

Reply:
Done both.